spec/components/layout_header_spec.rb

```diff
@@ -41,11 +41,13 @@ def component_name
     navigation_items = [
       { text: "Foo", href: "/foo", active: true },
       { text: "Bar", href: "/bar" },
+      { text: "Hello", href: "/hello", show_only_in_collapsed_menu: true },
```

Comment:
Wondering about this option name... would it be better as `show_only_in_mobile_menu`? I'm not sure.